.github/workflows/docusaurus.yaml

```diff
@@ -4,6 +4,9 @@ on:
   push:
     branches:
       - master
+  pull_request:
+    types:
+      - opened
```

Comment:
Think we also need `edited` and `reopened` ?

or remove `types` and set `branches` as per the `test` workflow, e.g.
```
  pull_request:
    branches:
      - master
      - 'release-*'
```

Reply:
I added the same cases for `push` and `pull_request` as they are in the `test` workflow.